packages/design-system/src/components/toggle-button.tsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * Implementation of the "Toggle Button" component from:
+ * https://www.figma.com/file/sfCE7iLS0k25qCxiifQNLE/%F0%9F%93%9A-Webstudio-Library?node-id=4-3199&t=lpT9jFuaiUnz1Foa-0
+ */
+
+import { forwardRef, type ComponentProps } from "react";
+import type { CSS } from "../stitches.config";
+import * as TogglePrimitive from "@radix-ui/react-toggle";
+import { IconButton } from "./icon-button";
+
+type Props = {
+  variant?: Extract<
+    ComponentProps<typeof IconButton>["variant"],
+    { "@initial"?: unknown }
+  >["@initial"];
```

Comment:
What does this code do? :)

I wouldn't use this weird `@initial` thing from stitches, would rather just define type as `"something" | "something1"` here